fetch.bs

```diff
@@ -2071,15 +2071,16 @@ for each associated <a for="fetch group">fetch record</a> whose
 <a for=connection>connection pool</a> consists of zero or more
 <dfn lt=connection export id=concept-connection>connections</dfn>. Each
 <a>connection</a> is identified by an <b>origin</b> (an
-<a for=/>origin</a>) and <b>credentials</b> (a boolean).
+<a for=/>origin</a>), <b>credentials</b> (a boolean), and
+<b>httpStatePartition</b> (a <a for=/>site</a> list or null).
```

Comment:
Does this mean that an httpStatePartition is a list of sites? May be add "key" to the name to be more clear that this is not the actual partition but the key "httpStatePartitionKey"?

Reply:
Done, everywhere (also "HTTP state partition" -> "HTTP state partition key")